src/lib/notificationWorker.js

```diff
@@ -18,35 +17,40 @@ const convertToExternalTransfer = require('../models/converters/transfers')
   .convertToExternalTransfer
 const getAffectedSubscriptions = require('../models/db/subscriptions')
   .getAffectedSubscriptions
+const getMatchingNotification = require('../models/db/notifications')
+  .getMatchingNotification
+const notificationDAO = require('../models/db/notifications')
```

Comment:
Not to be confused with "The DAO" https://daohub.org/ or "Dao" https://en.wikipedia.org/wiki/Tao or ... :)